server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailQueryMethod.scala

```diff
@@ -55,26 +55,22 @@ class EmailQueryMethod @Inject() (serializer: EmailQuerySerializer,
         .map(invocationResult => (invocationResult, processingContext)))
 
   private def processRequest(mailboxSession: MailboxSession, invocation: Invocation, request: EmailQueryRequest): SMono[Invocation] = {
-    val searchQuery = searchQueryFromRequest(request)
+    val comparators: List[Comparator] = request.comparator.getOrElse(Set(Comparator.default)).toList
+    val searchTerm: SearchQuery = new SearchQuery.Builder().sorts(comparators.map(comparator => comparator.toSort).asJava).build()
+    val searchQuery = MultimailboxesSearchQuery.from(searchTerm).build()
+
     SFlux.fromPublisher(mailboxManager.search(searchQuery, mailboxSession, Limit.default.value))
       .collectSeq()
       .map(ids => EmailQueryResponse(accountId = request.accountId,
         queryState = QueryState.forIds(ids),
         canCalculateChanges = false,
```

Comment:
Strong type canCalculateChanges

Reply:
This part is not on my PR, i will fix on separate commit.